src/main/java/com/buglife/sdk/LogMessage.java

```diff
@@ -62,4 +65,22 @@ int getContext() {
             return 0;
         }
     }
+
+    JSONObject toJSON() {
+        JSONObject json = new JSONObject();
+        Date timestamp = getTimestamp();
+
+        try {
+            json.put("type", getFormattedLevel());
+            json.put("message", getMessage());
+            json.put("context", getTag());
+
+            if (timestamp != null) {
+                json.put("timestamp", timestamp.getTime());
+            }
+        } catch (JSONException e) {
+            e.printStackTrace();
```

Comment:
I'd say swallowing this exception is fine in this case; not sure what else we can do here. :)